playbooks/vars/sample-variables-sap-swpm-default-mode-s4hana-distributed-aas-install.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+# Detect SAP NetWeaver software
+sap_install_media_detect_source_directory: "/software"
+sap_install_media_detect_source: local_dir
+sap_install_media_detect_swpm: true
+sap_install_media_detect_hostagent: true
+sap_install_media_detect_igs: true
+sap_install_media_detect_kernel: true
+sap_install_media_detect_webdisp: false
+
+# Product ID for New Installation
+sap_swpm_product_catalog_id: NW_DI:S4HANA2023.CORE.HDB.PD
+#NW_DI:S4HANA2023.CORE.HDB.PD   - Distributed AAS
+#NW_DI:S4HANA2023.CORE.HDB.ABAPHA   - High Availbility AAS
```

Comment:
To keep alignment with other samples, remove `   - High...` in-line comment as it is clear from suffix